script/Ciel/Day 5/05_01_CIEL05_1E.txt

```diff
@@ -672,7 +673,7 @@ Her long spear darts out at any Dead that dare approach, skewering them.
 [sha:7b5a697e75f8834d7089b30d6cc9825c3bb5774f]{
 -- Page 250, Offset 11519.
 -- その姿はどう見ても死者以上の“異常”だった。
-When it comes to being 'abnormal', she beats the Dead hands down.
+When it comes to being 'abnormal', she beats the Dead, hands down.
```

Comment:
I probably wouldn't add this second comma

Reply:
makes more sense in game because the linebreaks make it kinda awkward as is. I stand with the comma